README.md

```diff
@@ -27,6 +33,20 @@ If you're one of the brave few who wants to hack the firmware then read on.
 
 Going forward, it's easier to flash the firmware of your choice by using the downwards toolbar icon which is located rightwards of the *green play + toolbox icon*.
 
+### Minimal development setup
+
+1. Install the ARM cross-compiler, cross-assembler and stdlib implementation. Eg. on Arch Linux the packages `arm-none-eabi-binutils`, `arm-none-eabi-gcc`, `arm-none-eabi-newlib`.
+
+2. Install Node.js v12. If you have a later version, editing the version requirement in `lib/agent/package.json` *might* work.
+
+3. Build UHK Agent. `cd lib/agent && git pull --autostash && npm ci && npm run build`.
```

Comment:
`git clone --recursive` should pull Agent.

Reply:
Yes, the Agent repo should already be cloned but it might not be the latest commit. This was added because `make-release.js` also pulls the current master and I thought it might be useful.

Reply:
Now that I think about it, it's probably not a good idea to pull the master because it might be broken. We should stick to the Agent revision that the firmware repo points to, and change it explicitly if needed. Would you please modify the script accordingly?

Reply:
I did. Note that it's now possible to accidentally *not* update the submodule before building a release if submodules are not part of the typical workflow. I'm not sure what the best practices with submodules are.